src/Components/BrowsePage/BrowsePage.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import './BrowsePage.scss'
+import { CgUserlane } from "react-icons/cg";
+import { RiHomeSmileLine } from "react-icons/ri";
+import { Link } from 'react-router-dom';
+
+export const BrowsePage = (props) => {
+    const injectActivityInformation = () => {
+        return props.data.map((e, i) => {
+            return (
+                <li key ={i}className="hexa">
+                    <div className="inner-hexagon">
+                        <img className="hexa-image" 
+                        src={e.image_poster || e.image_100 || e.image} 
```

Comment:
I appreciate your effort in taking all of the different column names from our database's tables into account here.  Awesome for preventing so many errors for `undefined` values.

Reply:
more than preventing..., the app was crashing on me every time and made it painful at the beginning but I am glad it didn't take that much time to implement code for each different outcome.